### PR TITLE
[chore] add community files

### PR DESCRIPTION
**Pull request by jamescallumyoung**
Update the contibuting experience by adding a Contributing guide and issue templates.

- Add CONTRIBUTING
- Update issue templates

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+1. Go to '...'
+2. Click on '....'
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**System info**
```
Comment (vabatta) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: Are those necessary info for our application?

**Review by vabatta — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jamescallumyoung) [reply .github/ISSUE_TEMPLATE/bug_report.md:24]: Nope. I based this on another project 😆 I'll remove it now.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ - OS and version: [e.g. macOS Catalina (0.15.4)]
+ - Browser and version: [e.g. Firefox 80.0.1 (64-bit)]
+
```
Comment (jamescallumyoung) [inline .github/ISSUE_TEMPLATE/bug_report.md:27]:
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jamescallumyoung) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Nope, I based it on another project 😆 I'll remove it now.

**Review by vabatta — APPROVED** [review]

**Pull request merged**